03-getting-started/02-Configuration.adoc

```diff
@@ -10,101 +10,99 @@ toc::[]
+
+You can access configuration values via the **Config Provider**:
 
 [source, js]
 ----
 const Config = use('Config')
 const appSecret = Config.get('app.appSecret')
 ----
 
-The values are fetched using `Config.get()`. It takes as argument a string referencing the key you want, `fileName.key`. You can go deeper and fetch nested values using a dot-notation.
+Config values are fetched using `Config.get` which accepts a string argument referencing the key you want in the form `fileName.key`.
+
+You can fetch nested config values using dot notation:
 
 [source, js]
 ----
-// Example of a configuration - database.js
+// Example of a configuration file e.g. database.js
 {
   mysql: {
     host: '127.0.0.1',
   },
 }
 
-// You retrieve it using...
+// You can retrieve it like so...
 Config.get('database.mysql.host')
 ----
 
-If you aren't sure that the key is defined in your configuration you can supply a second argument that will be the default value.
+If you aren't sure that a key is defined in your configuration, you can supply a second argument that will be returned as the default value:
 
 [source, js]
 ----
 Config.get('database.mysql.host', '127.0.0.1')
 ----
 
-If you want to change the in-memory configuration values, you should make use of `Config.set()`.
+If you want to change the in-memory configuration values, use `Config.set`:
 
 [source, js]
 ----
 Config.set('database.mysql.host', 'db.example.com')
 ----
 
-WARNING: This only change the value in-memory. It will not write the new value on your config file.
+NOTE: `Config.set` will only change the value **in-memory**. It will not write the new value to your config file.
 
 == Env Provider
 
-When you are building an application, it happens quite often that you want to have a different configuration based upon the environment your code is running in.
+When building an application, you may want different configuration based upon the environment your code is running in.
 
-To fulfill this requirement, AdonisJs make use of the library link:https://github.com/motdotla/dotenv[dotenv, window="_blank"].
-Inside the root of every brand new AdonisJs project, you will find a `.env.example` file.
-If you used Adonis CLI to install your application, this file has been automatically copied to `.env`. Otherwise, you should copy it manually.
+To fulfill this requirement, AdonisJs uses the link:https://github.com/motdotla/dotenv[dotenv, window="_blank"] library.
 
-WARNING: The `.env` file should never be committed to your source control or shared with other people.
+Inside the root of every new AdonisJs project, you'll find an `.env.example` file.
+If you used AdonisJs CLI to install your application, this file will be automatically duplicated as `.env`. Otherwise, you should copy it manually.
 
-This file has a simple `key=value` syntax.
+NOTE: The `.env` file should never be committed to your source control or shared with other people.
 
-.env
+The `.env` file has a simple `key=value` syntax:
+
+..env
```

Comment:
The reason I changed `.env` to `..env` is so the example's title/filename renders correctly:

![env-filename-highlight](https://user-images.githubusercontent.com/1914481/47061845-100e6c80-d21f-11e8-87a3-d225811d8f31.png)

If I set it to `.env`, it would render as `env`, which is technically incorrect?